7-bank-project/2-forms/translations/assignment.fr.md

```diff
@@ -0,0 +1,13 @@
+# Concevez votre application bancaire
```

Comment:
What do you think about "Stylisez" instead of "Concevez"?

Reply:
Yes you're right "Stylisez" is better, I corrected it.
Thanks